pkgs/development/libraries/sqlite/default.nix

```diff
@@ -77,9 +68,21 @@ stdenv.mkDerivation rec {
     patchShebangs configure
   '';
 
-  configureFlags = [ "--enable-threadsafe" ] ++ lib.optional interactive "--enable-readline";
-
-  env.NIX_CFLAGS_COMPILE = toString ([
+  # sqlite relies on autosetup now; so many of the
+  # previously-understood flags are gone. They should instead be set
+  # on a per-output basis.
+  setOutputFlags = false;
+
+  configureFlags =
+    [
+      "--bindir=$(bin)/bin"
+      "--includedir=$(dev)/include"
+      "--libdir=$(out)/lib"
```

Comment:
You need to use `${placeholder "dev"}` when using `nix` context. Otherwise non-Makefile contexts do not expand this variable:

```
$ cat /nix/store/...-sqlite-3.49.2-dev/lib/pkgconfig/sqlite3.pc
...
includedir=$(dev)/include
...
```

This breaks packages like `rpm` as:

```
CMake Error in lib/CMakeLists.txt:
  Imported target "PkgConfig::SQLITE" includes non-existent path

    "$(dev)/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
```